src/main/target/common.h

```diff
@@ -17,12 +17,27 @@
  
 #pragma once
 
-#define CONFIG_FASTLOOP_PREFERRED_ACC 1
-
 #ifdef STM32F4
```

Comment:
Would it also make sense to have a common_stm32f[3|4].h included by the  targets headers to abstract this? This also would allow other #ifdef stm32f4 constants  in config.h to move into these headers?